soh/soh/Enhancements/controls/GameControlEditor.h

```diff
@@ -4,4 +4,4 @@ namespace GameControlEditor {
     static int CurrentPort = 0;
     static int BtnReading = -1;
     void Init();
-}
+}
```

Comment:
```suggestion
}

```
i think this should make it so we don't have a diff in this header file